metrics.md

```diff
@@ -0,0 +1,226 @@
+
+### `side_car_health_check_system_updates_total`
+
+This metric should be monotonically increasing. Specifically, the rate of this metric should be inversely correlated to the configured `UpdateInterval` in the oracle side-car configuration (`oracle.json`). To check this, you can run the following query in Prometheus:
+
+```promql
+rate(side_car_health_check_system_updates_total[5m])
+```
+
+![Architecture Overview](./resources/side_car_health_check_system_updates_total_rate.png)
+
+
+### `side_car_health_check_ticker_updates_total`
+
+This should be a monotonically increasing counter for each market. Each market's counter should be relatively close to the `side_car_health_check_system_updates_total` counter.
+
+To verify that the rate of updates for each market is as expected, you can run the following query in Prometheus:
+
+```promql
+rate(side_car_health_check_ticker_updates_total[5m])
+```
+
+![Architecture Overview](./resources/side_car_health_check_ticker_updates_total_rate.png)
+
+### `side_car_health_check_provider_updates_total`
+
+This metric should be monotonically increasing for each (provider, market) pair. To verify that the rate of updates for each provider is as expected, you can run the following query in Prometheus:
+
+```promql
+rate(side_car_health_check_provider_updates_total{provider="coinbase_api", success="true"}[5m])
+```
+
+![Architecture Overview](./resources/side_car_health_check_provider_updates_total_rate.png)
+
+### Health Metrics Summary
+
+In summary, the health metrics should be monitored to ensure that the side-car is updating its internal state, updating the price of each market, and fetching data from the price providers as expected. The rate of updates for each of these metrics should be inversely correlated with the `UpdateInterval` in the oracle side-car configuration.
+
+## Prices Metrics
+
+The side-car exposes various metrics related to market prices. These metrics are useful for monitoring the health of the price feeds and the aggregation process.
+
+### Price Feed Metrics
+
+The following price feed metrics are available to operators:
+
+* [`side_car_provider_price`](#side_car_provider_price): The last recorded price for a given price feed.
+* [`side_car_provider_last_updated_id`](#side_car_provider_last_updated_id): The last UNIX timestamp for a given price feed.
+
+#### `side_car_provider_price`
+
+This metric represents the last recorded price for a given price feed. The metric is indexed by the provider and market. For example, if we want to check the last recorded price of the BTC-USD market from the Coinbase API, we can run the following query in Prometheus:
+
+```promql
+side_car_provider_price{provider="coinbase_api", id="btc/usd"}
+```
+
+![Architecture Overview](./resources/side_car_provider_price_coinbase.png)
+
+Alternatively, if we wanted to check that last recorded price of the BTC-USD market across all price providers, we can run the following query in Prometheus:
```

Comment:
```suggestion
Alternatively, if we wanted to check that last recorded price of the BTC-USD market aggregated across all price providers, we can run the following query in Prometheus:
```

Reply:
or is this not true?

Reply:
how does this differ from `side_car_aggregated_price`?

Reply:
updated, this per provider price feeds.